scripts/setup_local.sh

```diff
@@ -3,7 +3,6 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-set -euo pipefail
```

Comment:
If it's designed to be  used through `source` command (or `.`), I think it is better to remove it.
If some command fails after running `. scripts/setup_local`, the shell crashes